docs/guides/migrate-from-buidler.md

```diff
@@ -0,0 +1,145 @@
+
+```js
+usePlugin("@nomiclabs/buidler-ethers");
+```
+
+you need to replace it with
+
+```js
+require("@nomiclabs/buidler-ethers");
+```
+
+Or, if you are using TypeScript, with
+
+```ts
+import "@nomiclabs/buidler-ethers";
+```
+
+### Configuring Hardhat Network
+
+Buidler EVM is now Hardhat Network, so if you are customizing it using the `buidlerevm` network config field,
+you need to rename it to `hardhat`. You can learn more about how to customize it, including enabling the Mainnet Forking
+functionality, [here](./config/README.md#hardhat-network).
+
+### Updating your Solidity config
+
+Hardhat has native support for multiple versions of Solidity. This requires a few changes in how you customize your
+`solc` setup.
+
+If you had something like this in your config:
+
+```js
+solc: {
+  version: "0.7.1",
+  optimizer: {
+    enabled: true
+  }
+}
+```
+
+You should change it to this:
+
+```js
+solidity: {
+  version: "0.7.1",
+  settings: {
+    optimizer: {
+      enabled: true
+    }
+  }
+}
+```
+
+This is a very simple config, but Hardhat supports arbitrarily complex compilation setups,
+down to the individual file level. Take a look at [this guide](./compile-contracts.md) to learn more about it.
+
+## TypeScript support changes
+
+Hardhat enables its TypeScript support when your config is written in TypeScript and ends in `.ts`. If you were using
+TypeScript, but a JavaScript config, please take a look at [this guide](./typescript.md).
+
+You don't need a `tsconfig.json` file with Hardhat. We recommend you delete it.
+
+If you prefer to keep it, you should base it on the template presented [here](./typescript.md#). Just make
+sure you removed all the `type-extension.d.ts` files, and keep your config file in your `files` field.
+
+## Importing artifacts and ambiguous names
+
+Hardhat supports multiple contracts with the same name.
+
+If you have contracts which share their name, you can't import their artifacts using just the name. 
+
+For example, if you have a contract named `Ownable`, and one of your dependencies has a contract with the same name, you won't be able to do
+`artifacts.require("Ownable")` nor `ethers.getContractFactory("Ownable")`. You need to use the contract's Fully
+Qualified Name instead.
+
+If you try to import a contract with a repeated name, Hardhat will fail, with an error that includes the different
+options to fix it. All you need to do is copy & paste them.
+
+## Mocha and VSCode setup changes
+
+If you have are running your tests directly with Mocha, or though a VSCode Mocha plugin, please take a look at [this
+updated guide](./vscode-tests.md).
+
+## Community plugins
+
+All of the official Buidler plugins have already been migrated to Hardhat.
+
+Some community-built plugins, haven't been migrated yet.
+
+### buidler-deploy
+
+This plugin is already being ported to Hardhat, and close to be released.
+
+Join our `#buidler-deploy` channel on [Discord](/discord) to get news and help about it.
```

Comment:
```suggestion
Join our `#hardhat-deploy` channel on [Discord](/discord) to get news and help about it.
```

Reply:
Maybe? Or maybe we'll rename the channel after it's published, idk.

Reply:
It should be buidler-deploy until migrated.